projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -257,6 +257,7 @@ public function add_options_menu() {
 		add_submenu_page( $slug, esc_attr__( 'Security', 'jetpack' ), __( 'Security', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/security/' . $this->domain );
 		add_submenu_page( $slug, esc_attr__( 'Performance', 'jetpack' ), __( 'Performance', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/performance/' . $this->domain );
 		add_submenu_page( $slug, esc_attr__( 'Writing', 'jetpack' ), __( 'Writing', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/writing/' . $this->domain );
+		add_submenu_page( $slug, esc_attr__( 'Reading', 'jetpack' ), __( 'Reading', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/reading/' . $this->domain );
```

Comment:
If I am not mistaken, the translation for the `Reading` string does not exist in the Jetpack plugin (I checked at https://translate.wordpress.org/projects/wp-plugins/jetpack/).

I wonder if this gets translated on time if the PR is merged before the code freeze. Will check with the Jetpack Crew.

Reply:
It's going to take a couple of weeks for the translations to come in after the Beta release. +i18n could tell you more about the exact schedule.

Reply:
Thanks, Jeremy! So the translation should be ready for the official release, right? I can also check with +i18n as you suggested.

Reply:
The official release happens one week after the Beta, so it will most likely not be ready in time. But +i18n can confirm that!

Reply:
Thank you! 🙂